pkg/app/web/src/components/settings-page/project/components/static-admin-form/index.tsx

```diff
@@ -102,7 +103,7 @@ const StaticAdminDialog: FC<{
             color="primary"
             disabled={
               formik.isValid === false ||
-              formik.values.username === currentUsername
+              formik.values.username === ""
```

Comment:
```suggestion
```
I think this line is unnecessary because already validated here.
https://github.com/pipe-cd/pipe/pull/2209/files#diff-0e1fa1872588c27aac2a8c319a43642fd2a980e41ced4719c3d2c96fc00ac622R43

Reply:
You are totally right! Let me remove this line.

Reply:
For sure!